pages/requests/[uuid].js

```diff
@@ -136,13 +173,9 @@ const Request = () => {
           <CollapsibleSection header='Additional Information' description={request.htmlDescription} />
           <Title addClass='mt-4' title='Documents' size='small' />
           {documents.length ? documents.map((document, index) => (
-            <Document 
+            <Document
               addClass='mt-3'
-              acceptSOW={acceptSOW(
-                request,
-                document.sowID,
-                accessToken,
-              )}
+              acceptSOW={() => handleAcceptingSOWandCreatingPO(document)}
```

Comment:
currently the `getAllPOs` function below is called nonstop for 2 reasons. this is the **first** reason.
the function was automatically being called, whether or not "accept sow" was pressed. (see attached video) 

https://user-images.githubusercontent.com/29032869/228001306-9da479e5-254a-4644-bb80-8e98dd2b9f8c.mp4